data/json/item_groups.json

```diff
@@ -6855,10 +6855,15 @@
     "id" : "helicopter",
     "magazine": 100,
     "items":[
+      { "group": "guns_launcher_milspec", "prob": 10 },
+      { "group": "guns_rifle_milspec", "prob": 10 },
+      { "item": "human_flesh", "prob": 4, "count-min": 1, "count-max": 10 },
+      { "item": "bone_human", "prob": 10, "count-min": 1, "count-max": 5 },
+      { "item": "rag_bloody", "prob": 10, "count-min": 1, "count-max": 40 },
```

Comment:
40 rags it's more than required for a blanket. Are you sure you want spawn that much rags?

Reply:
Yes. The maximum spawn is still less than the total butcher yield of a full set of army clothes+armor x 8 (2 pilots+6 soldiers)

Reply:
Gathered in a single stack by someone? You can move it in a separate item group and spawn in smaller bunches, 40 rags in one stack is really weird.

Reply:
Would reducing the count by 1/4th be okay?

Reply:
Would reducing the count to 1/4th be okay?

Reply:
Yes.

Reply:
Done. Do I need to make any other changes?